Sources/NFCPassportReader/PassportReader.swift

```diff
@@ -89,15 +91,24 @@ public class PassportReader : NSObject {
     public func overrideNFCDataAmountToRead( amount: Int ) {
         dataAmountToReadOverride = amount
     }
-    
-    public func readPassport( mrzKey : String, tags : [DataGroupId] = [], skipSecureElements : Bool = true, skipCA : Bool = false, skipPACE : Bool = false, useExtendedMode : Bool = false, customDisplayMessage : ((NFCViewDisplayMessage) -> String?)? = nil) async throws -> NFCPassportModel {
-        
+
+    public func readPassport(
+        mrzKey : String,
+        tags : [DataGroupId] = [],
+        skipSecureElements : Bool = true,
+        skipCA : Bool = false,
+        skipPACE : Bool = false,
+        useExtendedMode : Bool = false,
+        ignoreDataGroupParseErrors : Bool = false,
```

Comment:
Is there a way to make this with less changes: for example checking on iOS version internally and disabling logs in that way, instead of exposing the option in here